nextflow.config

```diff
@@ -118,6 +118,7 @@ params {
     config_profile_contact     = null
     config_profile_url         = null
     config_profile_name        = null
+    test_data_base             = 'https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq3'
```

Comment:
and since it is also set here already

Reply:
Also `https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq3` is different from the modules test-datasets